dist/main.js

```diff
@@ -1,121 +1,25 @@
-let isSorting = false;
-
-let sleep = milliSeconds => {
-  console.log('sleeping for ' + milliSeconds);
-  return new Promise(resolve => setTimeout(resolve, milliSeconds));
-};
-
-let addCssClass = (element, className) => {
-  element.classList.add(className);
-};
-
-let createDivElement = () => {
-  return document.createElement('DIV');
-};
-
-let getRandomHeight = () => {
-  let randomSum = Math.floor(10 + Math.random() * 51);
```

Comment:
Why do we need the random sum?

Reply:
I added it to make the output even more random. But I think we need to find a better way of generating the random numbers so that we have better inputs.